app/src/main/java/com/codesoom/assignment/exception/CusomResponseEntityExceptionHandler.java

```diff
@@ -0,0 +1,44 @@
+package com.codesoom.assignment.exception;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.HttpRequestMethodNotSupportedException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.Date;
+
+@RestController
+@ControllerAdvice
+public class CusomResponseEntityExceptionHandler extends ResponseEntityExceptionHandler {
+
+    @ExceptionHandler(TaskIdNotFoundException.class)
+    protected final ResponseEntity<Object> handleTaskIdNotFoundException
+            (Exception ex, WebRequest request) {
+        ExceptionResponse exceptionResponse
+                = new ExceptionResponse(new Date(),ex.getMessage(), request.getDescription(false));
+        return new ResponseEntity(ex.getMessage(), HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler(IllegalArgumentException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    protected final ResponseEntity<Object> handleIllegalArgumentException
+            (Exception ex, WebRequest request) {
+        ExceptionResponse exceptionResponse
+                = new ExceptionResponse(new Date(),ex.getMessage(), request.getDescription(false));
+        return new ResponseEntity<>(ex.getMessage(), HttpStatus.BAD_REQUEST);
+    }
```

Comment:
```suggestion
    @ExceptionHandler(IllegalArgumentException.class)
    @ResponseStatus(HttpStatus.BAD_REQUEST)
    protected final ExceptionResponse handleIllegalArgumentException (Exception ex, WebRequest request) {
        return new ExceptionResponse(new Date(),ex.getMessage(), request.getDescription(false));
    }
```

`@ResponseStatus(HttpStatus.BAD_REQUEST)`로 이미 응답 상태 코드는 정의하고 있으므로, ResponseEntity는 사용하지 않아도 괜찮을 것 같습니다

Reply:
제거 해 놓았습니다.